services/deploy/heroku.js

```diff
@@ -66,13 +66,13 @@ function Heroku(config) {
       });
   };
 
-  this.updateForestSecretKey = (appName, secretKey) => {
+  this.updateForestEnvSecret = (appName, envSecret) => {
```

Comment:
Where is the method that set the `FOREST_AUTH_SECRET` while deploying on heroku?